cmake/features.cmake

```diff
@@ -5,6 +5,16 @@
 #
 # Common cmake options
 #
+ie_option (ENABLE_MAC_X86_EMULATION "Enable x86_64 architectures emulation for macOS" OFF)
+if (ENABLE_MAC_X86_EMULATION)
+    if (APPLE)
+        set(CMAKE_OSX_ARCHITECTURES "x86_64")
```

Comment:
let's use this flag as public one.
We need to support values:
- x86_64
- arm
- x86_64;arm

Reply:
arm64 and x86_64 works separately.
Universal binaries don't work because some dependencies don't have symbols for universal binaries.

Reply:
what's about functional tests?